apps/strapi-cms/config/middlewares.ts

```diff
@@ -1,6 +1,31 @@
-export default [
+export default ({ env }: any) => [
   'strapi::errors',
-  'strapi::security',
+  {
+    name: 'strapi::security',
+    config: {
+      contentSecurityPolicy: {
+        useDefaults: true,
+        directives: {
+          'connect-src': ["'self'", 'https:'],
+          'img-src': [
+            "'self'",
+            'data:',
+            'blob:',
+            'dl.airtable.com',
+            env('AWS_BUCKET_FULL_URL'),
```

Comment:
Can we use the CDN path instead the bucket path?

Reply:
No idea atm.

@maurodandrea is this something we can test in the short term?

Otherwise I would just pin this issue, merge the PR and create another for this specific change.

Reply:
I can pass you the CDN URL of the S3 bucket in a var of the task definition. How should the var be named for Strapi, CDN_BASE_URL ?

Reply:
following Alessandro's instructions, we leave the AWS_BUCKET_FULL_URL